src/handlers/next/saleor-webhooks/process-saleor-webhook.test.ts

```diff
@@ -173,26 +172,4 @@ describe("processAsyncSaleorWebhook", () => {
       schemaVersion: null,
     });
   });
-
-  it("Return schema version if saleor-schema-version header is present", async () => {
```

Comment:
I had a problem with this test - currently `version` will be on payload but we mock full `raw-body` in the top of the tests. I tried:
1. Removing mock and passing values directly - I've got a timeouts
2. Using technique from https://github.com/vitest-dev/vitest/discussions/4124 to dynamically asssign value of the mock per test

All without luck - do you have clue what can be done here?

Reply:
i dont understand. header will be gone, so this test should be removed. 
you should have test that mocks graphql body and checks if it extracts it, right?